regimes/pl/invoices.go

```diff
@@ -27,7 +27,7 @@ func (v *invoiceValidator) validate() error {
 		// Rectification state determined by Preceding value.
 		validation.Field(&inv.Type, validation.In(
 			bill.InvoiceTypeStandard,
-			// bill.InvoiceTypeCorrective,
+			bill.InvoiceTypeCorrective,
```

Comment:
So my initial impression here is that these in fact should be "Credit Notes" as opposed to Corrective invoices. This is a confusing subject as some countries, like Spain, don't recognise the "credit notes" as such, but instead define the concept of corrective invoices and a replacement strategy.

From the looks of things here and the example with negative values, I'd suggest using the `bill.InvoiceTypeCreditNote` type instead.

The difference in GOBL is as follows:
 * Corrective invoices *replace* the previous document.
 * Credit (or Debit) notes *extend* the previous document.

It may indeed be that Poland supports *both* types.

Reply:
It definitely extends the previous invoice. I'll change the type.